src/Main/main.cpp

```diff
@@ -67,9 +67,9 @@ int main(int argc, char** argv) {
     std::filesystem::path yosysPath = binpath / "yosys";
     std::filesystem::path vprPath = binpath / "vpr";
     std::filesystem::path openFpgaPath = binpath / "openfpga";
-    std::filesystem::path pinConvPath = binpath / "stars";
+    std::filesystem::path pinConvPath = binpath / "planning";
```

Comment:
Would you like to fix the variable name here ? It's confusing to see a pinConvPath and startsPath to be the same.